osmaxx/geodesy/coordinate_reference_system.py

```diff
@@ -0,0 +1,99 @@
+from django.contrib.gis.geos.collections import MultiPolygon
+from django.contrib.gis.geos.polygon import Polygon
+
+from osmaxx.conversion_api.coordinate_reference_systems import WGS_84
+
+MIN_LONGITUDE_DEGREES = -180
+MAX_LONGITUDE_DEGREES = +180
+
+
+class UniversalTransverseMercatorZone:
+    HEMISPHERE_PREFIXES = dict(
+        north=326,
+        south=327,
+    )
+    NUMBER_OF_ZONES_PER_HEMISPHERE = 60
+    VALID_ZONE_NUMBERS = range(1, NUMBER_OF_ZONES_PER_HEMISPHERE + 1)
+    ZONE_WIDTH_DEGREES = 360 / NUMBER_OF_ZONES_PER_HEMISPHERE
+    MAX_LONGITUDE_OFFSET = 90.0 - 9.9e-14
```

Comment:
This isn't something I can understand by just seeing the value.
I think this would be one place where a comment wouldn't hurt (since putting the entire explanation into a variable seems to generate a bit a longish name :laughing: ).